openmetadata-ui/src/main/resources/ui/playwright/utils/lineage.ts

```diff
@@ -0,0 +1,386 @@
+  );
+  await page.locator('.edge-info-drawer').isVisible();
+  await page
+    .locator('.edge-info-drawer [data-testid="Edge"] a')
+    .filter({ hasText: pipelineData.name });
+
+  await page.click('.edge-info-drawer [data-testid="edit-description"]');
+  await page.locator('.ProseMirror').first().click();
+  await page.locator('.ProseMirror').first().clear();
+  await page.locator('.ProseMirror').first().fill(description);
+  const descRes = page.waitForResponse('/api/v1/lineage');
+  await page.getByTestId('save').click();
+  await descRes;
+
+  await expect(
+    page.getByTestId('asset-description-container').getByRole('paragraph')
+  ).toContainText(description);
+};
+
+const verifyPipelineDataInDrawer = async (
+  page: Page,
+  fromNode: EntityClass,
+  toNode: EntityClass,
+  pipelineItem: PipelineClass,
+  bVerifyPipelineLineage: boolean
+) => {
+  const fromNodeFqn = get(fromNode, 'entityResponseData.fullyQualifiedName');
+  const toNodeFqn = get(toNode, 'entityResponseData.fullyQualifiedName');
+  const pipelineName = get(pipelineItem, 'entityResponseData.name');
+
+  await page.click(
+    `[data-testid="pipeline-label-${fromNodeFqn}-${toNodeFqn}"]`
+  );
+
+  await page.locator('.edge-info-drawer').isVisible();
+  await page
+    .locator('.edge-info-drawer [data-testid="Edge"] a')
+    .filter({ hasText: pipelineName });
+
+  if (bVerifyPipelineLineage) {
+    await page.locator('.edge-info-drawer [data-testid="Edge"] a').click();
+    await page.click('[data-testid="lineage"]');
+    await fromNode.visitEntityPage(page);
+    await page.click('[data-testid="lineage"]');
+  } else {
+    await page.click('.edge-info-drawer .ant-drawer-header .anticon-close');
+  }
+};
+
+export const applyPipelineFromModal = async (
+  page: Page,
+  fromNode: EntityClass,
+  toNode: EntityClass,
+  pipelineItem?: PipelineClass
+) => {
+  const fromNodeFqn = get(fromNode, 'entityResponseData.fullyQualifiedName');
+  const toNodeFqn = get(toNode, 'entityResponseData.fullyQualifiedName');
+  const pipelineName = get(pipelineItem, 'entityResponseData.name');
+  const pipelineFqn = get(
+    pipelineItem,
+    'entityResponseData.fullyQualifiedName'
+  );
+
+  await page.click(`[data-testid="edge-${fromNodeFqn}-${toNodeFqn}"]`, {
+    force: true,
+  });
+
+  await page.click('[data-testid="add-pipeline"]');
+  const field = await page.locator(
+    '[data-testid="add-edge-modal"] [data-testid="field-input"]'
+  );
+  await field.click();
+  await field.fill(pipelineName);
+
+  await page.click(`[data-testid="pipeline-entry-${pipelineFqn}"]`);
+
+  const saveRes = page.waitForResponse('/api/v1/lineage');
+  await page.click('[data-testid="save-button"]');
+  await saveRes;
+};
+
+export const deleteNode = async (page: Page, node: EntityClass) => {
+  const nodeFqn = get(node, 'entityResponseData.fullyQualifiedName');
+  await page
+    .locator(`[data-testid="lineage-node-${nodeFqn}"]`)
+    .dispatchEvent('click');
```

Comment:
I have seen this clicking method to lot of places, is there any use case for that?

Reply:
Kept it explicitly. Refer this 
https://playwright.dev/docs/input#programmatic-click